src/main/java/frc/team2412/robot/Subsystems/ControlPanelColorSubsystem.java

```diff
@@ -0,0 +1,99 @@
+package frc.team2412.robot.Subsystems;
+
+import com.revrobotics.ColorMatch;
+import com.revrobotics.ColorMatchResult;
+import com.revrobotics.ColorSensorV3;
+
+import edu.wpi.first.wpilibj.Talon;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.team2412.robot.Subsystems.constants.ControlPanelConstants;
+
+public class ControlPanelColorSubsystem extends SubsystemBase {
+
+	private ColorSensorV3 m_colorSensor;
+	private ColorMatch m_colorMatcher;
+	private Talon m_wheelMotor;
+	private Color m_CurrentColor;
+	private Color m_StartColor;
+	private Color m_ColorUnderBar;
+	private int rotationCount = 0;
+
+	public ControlPanelColorSubsystem(ColorSensorV3 colorSensor, Talon motor, ColorMatch colorMatch) {
+		this.m_colorSensor = colorSensor;
+		this.m_wheelMotor = motor;
+		this.m_colorMatcher = colorMatch;
+	}
+
+	public void rotateControlPanel() {
+		rotationCount = 0;
+		m_StartColor = m_colorSensor.getColor();
+		m_wheelMotor.set(0.5);
+		while (rotationCount <= 7) {

```

Comment:
Ok, this looks good... BUT two things, have the command end the motor, not having it be controlled by a loop. also, make the rotationCount a constant